src/assets/js/fields/color-picker.js

```diff
@@ -4,7 +4,12 @@
 		$( '.notification-color-picker:visible' ).wpColorPicker();
 	} );
 
-	notification.hooks.addAction( 'notification.carrier.repeater.row.added', function( $cloned ) {
-		$cloned.find( '.notification-color-picker' ).wpColorPicker();
+	notification.hooks.addAction( 'notification.carrier.repeater.row.added', function( instance ) {
```

Comment:
`instance` doesn't tell much. Be as much specific as possible, `repeater` would be better